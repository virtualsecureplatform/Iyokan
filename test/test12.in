[[rom]]
name = "rom"
size = 4096           # 512 * 8
bytes = [0, 1, 2, 3, 4, 5, 6, 7]

[[bits]]
name = "addr"
size = 7
bytes = [1]
