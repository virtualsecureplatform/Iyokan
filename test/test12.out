rom = []
ram = []
cycles = 1
bits = [
{bytes=[4,5,6,7],size=32,name="rdata"},
]
